jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/AsyncTaskTransactionTest.java

```diff
@@ -154,4 +156,20 @@ private KieSession registerAsyncHandler(KieSession ksession) {
         return ksession;
     }
 
+    @Override
+    protected PoolingDataSource setupPoolingDataSource() {        
+        
+        Properties dsProps = PersistenceUtil.getDatasourceProperties();
+        String jdbcUrl = dsProps.getProperty("url");
+        String driverClass = dsProps.getProperty("driverClassName");        
+
+        // Setup the datasource
+        PoolingDataSource ds1 = PersistenceUtil.setupPoolingDataSource(dsProps, "jdbc/jbpm-ds", false);
+        if (driverClass.startsWith("org.h2")) {
+            ds1.getDriverProperties().setProperty("url", jdbcUrl);
+        }
+        ds1.getDriverProperties().setProperty("POOL_CONNECTIONS", "false");
```

Comment:
Shouldn't we make this default until there is a real pooling datasource?

Reply:
But maybe it is not worth it if other tests are OK :)

Reply:
that's exactly the issue :)